prow/github/client.go

```diff
@@ -269,6 +269,7 @@ type Client interface {
 	UserClient
 	HookClient
 	ListAppInstallations() ([]AppInstallation, error)
+	IsAppInstalled(org, repo string) (bool, error)
```

Comment:
As an alternate, ListAppInstallations is pretty much guaranteed to be cached at all times  so looking it up from there will save you a request to github (but not opposed to this change, just mentioning it)

Reply:
Unfortunately, it doesn't actually have info about individual repo installs in there. We can only check if it is installed for the entire org or if repos were selected, but not which ones.

Reply:
oh wow indeed, that is not great.
/lgtm
/approve
/retest